crates/cli-support/src/wit/mod.rs

```diff
@@ -1479,16 +1479,17 @@ version of wasm-bindgen that uses a different bindgen format than this binary:
      this binary schema version: {}
```

Comment:
```suggestion
     this binary schema version: {my_version}
```
Let's use the new syntax and remove the parameters.